tests/test_main.py

```diff
@@ -0,0 +1,12 @@
+from cookiecutter.main import is_repo_url
+
+
+def test_is_repo_url():
+    """Verify is_repo_url works."""
+    assert is_repo_url('gitolite@server:team/repo') is True
+    assert is_repo_url('git@github.com:audreyr/cookiecutter.git') is True
+    assert is_repo_url('https://github.com/audreyr/cookiecutter.git') is True
+    assert is_repo_url('gh:audreyr/cookiecutter-pypackage') is True
```

Comment:
I am afraid this is not a test for ``mercurial``, but for the abbreviations.

I was thinking of testing something like ``https://bitbucket.org/pokoli/cookiecutter-trytonmodule.hg`` :no_mouth: